### PR TITLE
chore(github-template): update issue template

### PR DESCRIPTION
**Pull request by daern91**

### CODE REVIEW

**Review by wizzy25 — APPROVED** [review]

File: .github/ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -1,3 +1,6 @@
+### Support reference number
+<!-- Reference to JIRA ticket issue if available. -->
```
Comment (adnasa) [inline .github/ISSUE_TEMPLATE.md:2]: We probably shouldn't encourage colleagues to post JIRA links on open source projects? this doesn't make sense right now 😛 
cc @wizzy25

**Review by adnasa — CHANGES_REQUESTED** [review]

File: .github/ISSUE_TEMPLATE.md (hunk 1 above)
Comment (wizzy25) [reply .github/ISSUE_TEMPLATE.md:2]: @adnasa This has been discussed severally. The JIRA links will not be accessible to the public. And not all issues have accompanying JIRA references

Comment (adnasa) [reply .github/ISSUE_TEMPLATE.md:2]: i don't think that really justifies it. my point is that JIRA shouldn't even openly linked in the first place.

Comment (adnasa) [reply .github/ISSUE_TEMPLATE.md:2]: but if you guys really need this and it's fine with your PO, then all good from my side

**Pull request merged**